recipes.js

```diff
@@ -1,11 +1,83 @@
 const mongoose = require('mongoose');
 const Schema   = mongoose.Schema;
 const data = require('./data.js')
+var currentDate = new Date();
+
+const recipeSchema = new Schema({
+  title: {
+    type: String,
+    required: true, 
+    unique: true,
+  },
+  level: {
+    type: String, 
+    enum: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"],
+  },
+  ingredients: [String],
+  cousine: {
+    type: String, 
+    required: true,
+  },
+  dishType: {
+    type: String, 
+    enum: ["Breakfast", "Dish", "Snack", "Drink", "Dessert", "Other"],
+  },
+  image: {
+    type: String, 
+    default: "https://images.media-allrecipes.com/images/75131.jpg",
+  },
+  duration: {
+    type: Number,
+    min: 0,
+  },
+  creator: String,
+  created: {
+    type: Date, 
+    default: currentDate.toDateString(),
+  }
+});
+
+const Recipe = mongoose.model("Recipe", recipeSchema);
+
+Recipe.create({title: "Poulet à la moutarde", level: "Easy Peasy", ingredients: ["Poulet", "Moutarde", "Oignons", "Sel", "Poivre"], cousine: "French", dishType: "Dish", image: "https://static.cuisineaz.com/610x610/i89759-poulet-a-la-moutarde-au-four.jpg", min: 60, creator: "Chef Nerimene" })
```

Comment:
min is not a field, the field is "duration", right now the creation of the recipe works because the duration field is not required, so it creates the recipe not taking into account the "min: 60" part, but if you replace 'min' by duration' here, you will have a proper recipe in your database :)